pynicotine/gtkgui/ui/popovers/wishlist.ui

```diff
@@ -24,7 +24,7 @@
           <object class="GtkEntry" id="wish_entry">
             <property name="visible">1</property>
             <property name="hexpand">1</property>
-            <property name="placeholder-text" translatable="yes">Add search term...</property>
+            <property name="placeholder-text" translatable="yes">Add Wish...</property>
```

Comment:
- Fixed: Changed placeholder text to "Add Wish..." because "Add search term..." was ambiguous with main search.